package.json

```diff
@@ -11,6 +11,7 @@
   "main": "umd/skatejs.js",
   "module": "es/index.js",
   "es2015": "es-latest/index.js",
+  "esnext": "src/index.js",
```

Comment:
What do you think about:

1. Remove the `es2015` key above.
2. Point this to `esnext/index.js`.
3. Update the build command to use `esnext` instead of `es-latest`

?